packages/contracts-bedrock/scripts/go-ffi/differential-testing.go

```diff
@@ -366,37 +366,39 @@ func DiffTestUtils() {
 		if len(args) != 3 && len(args) != 5 && len(args) != 7 {
 			panic("Error: cannonMemoryProofWithProof requires 2, 4, or 6 arguments")
 		}
-		pc, err := strconv.ParseUint(args[1], 10, 32)
+		pc, err := strconv.ParseUint(args[1], 10, arch.WordSize)
 		checkErr(err, "Error decoding addr")
-		insn, err := strconv.ParseUint(args[2], 10, 32)
+		insn, err := strconv.ParseUint(args[2], 10, arch.WordSize)
```

Comment:
(nit) We're accepting a `Word` for the`insn` param even though the instruction is always 32-bit.  Maybe we should just change the language here to label pc/insn as a generic memAddr/memValue pair.  Or else, keep `insn` 32-bit and transform insn into a Word value before setting.

Reply:
good point. Will do.